src/StatusQ/Controls/StatusSlider.qml

```diff
@@ -19,24 +19,27 @@ Slider {
         implicitWidth: 100
         implicitHeight: 4
         height: implicitHeight
-        color: {
-            if (statusSlider.value === statusSlider.to) {
-                return Theme.palette.primaryColor1
-            }
-            return Theme.palette.primaryColor3
-        }
+        color: Theme.palette.primaryColor3
         radius: 2
+
+//        Rectangle {
```

Comment:
Do you plan to reuse it in a future update of this PR? For sure, we don't want to keep commented dead code around.